### PR TITLE
Add v8.2 for search

### PR DESCRIPTION
**Pull request by qiancai**

### CODE REVIEW

File: docsearch/algolia_configs/latest_commit.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "en-tidb-dev": "9680a4b7e1e708f90341dcbb0795247e64ba441a",
+    "en-tidb-v8.2": "to-be-updated",
```
Comment (qiancai) [inline docsearch/algolia_configs/latest_commit.json:3]: to-be-updated according to the latest commit in https://github.com/pingcap/docs/commits/release-8.2/

File: docsearch/algolia_configs/latest_commit.json [hunk 2]
```diff
@@ -26,6 +27,7 @@
     "en-tidb-v3.0": "ff3bdb392d243f7a48051e037fb750579e7a6ca5",
     "en-tidb-v2.1": "2755565832044b9a1fdc4be01057dc398098e6f3",
     "zh-tidb-dev": "f27485b695f5955aac3922b4b5e908d0950ca854",
+    "zh-tidb-v8.2": "to-be-updated",
```
Comment (qiancai) [inline docsearch/algolia_configs/latest_commit.json:30]: to-be-updated according to the latest commit in https://github.com/pingcap/docs/commits/release-8.2/

**Review by Yuiham — COMMENTED** [review]
Please remember to update the commit SHA. Rest LGTM

File: docsearch/algolia_configs/latest_commit.json (hunk 2 above)
Comment (qiancai) [inline docsearch/algolia_configs/latest_commit.json:30]:
```suggestion
    "zh-tidb-v8.2": "5c33ba2b6865802c843542294b24a193b0d51d77",
```

File: docsearch/algolia_configs/latest_commit.json (hunk 1 above)
Comment (qiancai) [inline docsearch/algolia_configs/latest_commit.json:3]:
```suggestion
    "en-tidb-v8.2": "f8c8a26fc8143f784101ca3d75a7275988a19da0",
```

**Review by Yuiham — APPROVED** [review]

**Pull request merged**